src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result = result - 50;
```

Comment:
Use a constant for the long-term discount threshold and the discount amount instead of hardcoding the values 7 and 50.